docs/proposals/20210310-opt-in-autoscaling-from-zero.md

```diff
@@ -204,8 +209,16 @@ status:
     memory: 500mb
     cpu: "1"
     nvidia.com/gpu: "1"
+  nodeInfo:
```

Comment:
i would just like to call attention to this, at it represents a growth in our API for the infrastructure templates.